src/usr/local/www/system_camanager.php

```diff
@@ -90,6 +90,7 @@
 	$name = $a_ca[$id]['descr'];
 	unset($a_ca[$id]);
 	write_config();
+	ca_trustedcerts_generate();
```

Comment:
Would rather trusted certs be generated/removed on a per cert add/edit basis instead of regenerating all of them every time a change is made.  Even changes that do not make changes to trusted certs settings.